_sass/base/_override.scss

```diff
@@ -56,7 +56,7 @@ pre, pre.prettyprint {
   background-color: $codeBkColor;
   color: #000000;
   border: none !important;
-  //font-size: 14px !important;
+  font-size: 10px !important;
```

Comment:
can we do this as a separate thing, and not as part of this PR? Because it affects all prettyprint content and we might want to revert it if stuff goes awry